lib/index-static.jsx

```diff
@@ -39,9 +39,10 @@ function generateWithPageHTML(url, options, pageHTML) {
           return <meta key={i} name={name} content={options.meta[name]}/>;
         })}
         <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,400italic,600,700,600italic,700italic,800,800italic"/>
+        <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css"/>
         <link rel="stylesheet" href="/vendor/bootstrap/css/bootstrap.min.css"/>
-        <link rel="stylesheet" href="/vendor/webmaker-app-icons/css/ionicons.min.css"/>
-        <link href="https://mozorg.cdn.mozilla.net/media/css/tabzilla-min.css" rel="stylesheet" />
+        <link rel="stylesheet" href="/vendor/font-awesome/css/font-awesome.min.css"/>
+        <link rel="stylesheet" href="https://mozorg.cdn.mozilla.net/media/css/tabzilla-min.css" />
```

Comment:
I swapped the ordering of `rel` and `href` to make this line more unified with other lines